example/Python/src/DigitalTwin/KeyFob/README.md

```diff
@@ -67,9 +67,9 @@ user@twin-vm ~ $ docker container attach CONTAINER_ID
 
 ### Running the local key fob
 
-Open another terminal in the directory where the `docker-compose.yml` is located. Run:
+Open another terminal in the directory where the `docker-compose.yml` is located. You might have to export `$RTI_IP` again. Run:
```

Comment:
I think this new sentence can be confusing because it's not clear what "again" here is referring to. I can see that `setup.sh` is supposed to export `RTI_IP`, so I'm assuming this sentence is a reference to that. But I think the script is just setting a local `RTI_IP` variable, not an environment variable.

It would also be helpful here I think if `-i` and `--host` are explained here as well.

Reply:
the user might have to export `RTI_IP` again because they opened a new terminal.

Reply:
Right, but exporting the `RTI_IP` environment variable is not mentioned earlier at all. This is the first time that it is mentioned as far as I can see, or am I missing a step?

Reply:
oh wait, `RTI_IP` is exported in `setup.sh`. I'll specify that.

Reply:
I added a script that the user can just run to both re-export `RTI_IP` and start the local copy